ansible/roles/offline-installer/templates/setupOfflineInstaller.sh.j2

```diff
@@ -5,24 +5,24 @@ npm install -g n
 n 8.16
 cd /project/src
 
-if [ "{{offline_installer_type}}" != "windows32" ];
+if [ "{{offline_installer_type}}" != "windows32bit" ];
 then
 #Build the offline installer
-	npm install
-	npm run dist
+        npm install
+        npm run dist
 fi
 
-if [ "{{offline_installer_type}}" == "windows32" ];
+if [ "{{offline_installer_type}}" == "windows32bit" ];
 then
-#	npm run dist
-	npm run dist-win32 
-	cp '/project/src/dist/1.0.0/win/ia32/DIKSHA Setup 1.0.0.exe' ../offline_artifacts/{{time}}/
-elif [ "{{offline_installer_type}}" == "windows64" ];
+#       npm run dist
+        npm run dist-win32
+        cp '/project/src/dist/1.0.0/win/ia32/DIKSHA Setup 1.0.0.exe' ../offline_artifacts/{{time}}/diksha_{{installer_version}}_windows32bit.exe
```

Comment:
references to diksha should not be present in sunbird code, @harishgilimi can you remove them in the code level in repo `https://github.com/Sunbird-Ed/sunbird-desktop-app`?

Reply:
This issue is resolved.